doc/api/net.md

```diff
@@ -722,25 +722,26 @@ that the
 [`socket.connect(options[, connectListener])`][`socket.connect(options)`]
 callback is a listener for the `'connect'` event.
 
-### `socket.destroy([exception])`
+### `socket.destroy([error])`
 <!-- YAML
 added: v0.1.90
 -->
 
-* `exception` {Object}
+* `error` {Object}
 * Returns: {net.Socket}
 
-Ensures that no more I/O activity happens on this socket. Only necessary in
-case of errors (parse error or so).
+Ensures that no more I/O activity happens on this socket. This destroys
+the stream but does not explicitly cause a TCP stream reset (RST).
```

Comment:
@lpinca: please note updated version, better?

Reply:
I would honestly not add those details that I think are confusing for most users. "Destroys the stream and closes the connection". Interested users can dig by themselves to see what happens under the hood.

Reply:
Changed to your suggestion.